### PR TITLE
Config files for multi-year, high-topped atmosphere simulation with clearksy radiation

### PR DESCRIPTION
**Pull request by akshaysridhar**
Linked Issue: #2060

### CODE REVIEW

**Comment by szy21** [comment]
The configuration file looks good to me. It might be good to change kappa_4 back to 1e16.

**Comment by akshaysridhar** [comment]
Screenshot 2023-09-26 at 1 53 59 PM
WIP. Time-averaging over last 200 days.

File: config/longrun_configs/longrun_aquaplanet_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+dz_top: 3000.0
+z_max: 55000.0
+rayleigh_sponge: true
+kappa_4: 2.0e16
+moist: "equil"
+precip_model: "0M"
+surface_setup: "DefaultMoninObukhov"
+vert_diff: "true"
+rad: "clearsky"
+job_id: "longrun_hs_rhoe_equilmoist_nz64_0M_55km_rs35km_clearsky"
```
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.yml:17]:
```suggestion
job_id: "longrun_aquaplanet_rhoe_equilmoist_nz64_0M_55km_rs35km_clearsky"
```

File: config/longrun_configs/longrun_aquaplanet_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+toml: [toml/longrun_hs_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.toml]
```
Comment (szy21) [inline config/longrun_configs/longrun_aquaplanet_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.yml:18]:
```suggestion
toml: [toml/longrun_aquaplanet_rhoe_equilmoist_nz63_0M_55km_rs35km_clearsky.toml]
```

**Comment by szy21** [comment]
The results look good. Could you run one with `idealized_insolation: false` as well? Just to make sure the insolation is working fine.

**Comment by akshaysridhar** [comment]
> The results look good. Could you run one with `idealized_insolation: false` as well? Just to make sure the insolation is working fine.

queued.

**Review by szy21 — APPROVED** [review]

**Pull request merged**